packages/server/src/validator/methods.ts

```diff
@@ -1,12 +1,13 @@
 export const METHODS = {
+  // TODO: Implement this check once the method is completely ready
```

Comment:
instead of removing the whole validation maybe remove the validation for `data` to unblock MM.

Add a ticket to come back to this later.

Then add a separate ticket to come back to `accessList` validation